test/src/engines/html5/html5.spec.js

```diff
@@ -31,6 +32,20 @@ describe('Html5', () => {
     engine._el.should.be.a('HTMLVideoElement');
   });
 
+  it('should prevent too short buffering event', done => {
+    const progressiveSource = sourcesConfig.Mp4.progressive[0];
+    const engine = Html5.createEngine(progressiveSource);
+    engine._eventManager.listen(engine, Html5EventType.WAITING, () => {
+      done();
```

Comment:
so wont the test finish after the first WAITING event ?

Reply:
w/o the fix the test will get multiple done calls

Reply:
I'm not sure I understand what is being tested - don't we want to check the number of WAITING events ?

Reply:
should be only one waiting because the shorter one should be ignored